LokiLabs/microsoft-graph-explorer-teams-app
# Localize All Text

Hello hello!

The PR is rather straightforward, I think ^^ Just an update for all of the content in interns/dev to become localized with the changes made via https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/15

**To test:**
See https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/15 for testing steps!

**Note:**
There is a bug with some of the RSC list not being localized. This is a separate matter, as the bug existed prior to this PR (See [AD#39416](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_backlogs/backlog/GI21%20-%20Graph%20Explorer/Backlog%20items/?workitem=39416))

--

*Ticket:*
[AD#39307](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_backlogs/backlog/GI21%20-%20Graph%20Explorer/Backlog%20items/?workitem=39307)
